kustomize/overlays/minikube/manager/kustomization.yaml

```diff
@@ -7,6 +7,13 @@ secretGenerator:
   - EVENT_BRIDGE_KAFKA_BOOTSTRAP_SERVERS=my-cluster-kafka-bootstrap.kafka:9092
   - EVENT_BRIDGE_KAFKA_CLIENT_ID=client-id
   - EVENT_BRIDGE_KAFKA_CLIENT_SECRET=secret
+  - EVENT_BRIDGE_KAFKA_SECURITY_PROTOCOL=SASL_SSL
```

Comment:
Please remove https://github.com/kostola/sandbox/blob/MGDOBR-225/kustomize/overlays/minikube/manager/patches/deploy-config.yaml#L13 to avoid value duplication.

Reply:
Done. Good catch @sutaakar :+1: